pysat/_files.py

```diff
@@ -412,54 +450,70 @@ def _store(self):
         if new_flag:
             if self.write_to_disk:
                 # Save the previous data in a backup file
-                prev_name = "_".join(["previous", stored_name])
-                stored_files.to_csv(os.path.join(self.home_path, prev_name),
+                prev_name = os.path.join(self.home_path, 'archive', stored_name)
+                stored_files.to_csv(prev_name,
                                     date_format='%Y-%m-%d %H:%M:%S.%f',
-                                    header=False)
+                                    header=[self.data_path])
 
                 # Overwrite the old reference file with the new file info
                 self.files.to_csv(os.path.join(self.home_path, stored_name),
                                   date_format='%Y-%m-%d %H:%M:%S.%f',
-                                  header=False)
+                                  header=[self.data_path])
             else:
                 # Update the hidden File attributes
                 self._previous_file_list = stored_files
                 self._current_file_list = self.files.copy()
+
         return
 
-    def _load(self, prev_version=False):
-        """Load stored filelist and return as Pandas Series
+    def _load(self, prev_version=False, update_path=True):
+        """Load stored filelist
 
         Parameters
         ----------
         prev_version : boolean
             if True, will load previous version of file list
+        update_path : boolean
+            If True, the path written to stored info will be
+            assigned to self.data_path. (default=True)
 
         Returns
         -------
         pandas.Series
-            Full path file names are indexed by datetime
-            Series is empty if there is no file list to load
+            file path names, indexed by datetime. Series is empty if no
```

Comment:
```suggestion
            File path names, indexed by datetime. Series is empty if no
```

Reply:
Also, it looks like the return variable now is `out`, so this needs to be updated further.

Reply:
I don't really understand the comment about `out`. There is a variable `out` in method below so I'll poke at that docstring.

Reply:
Could be that.  From the display it appeared to be a part of this method.

Reply:
Capitalization still stands, though.

Reply:
I accepted your suggestion.

Here's what my local system is showing,
```
        Returns
        -------
        pandas.Series
            File path names, indexed by datetime. Series is empty if no
            files are found.
```